### PR TITLE
Set liberry config

### LINKED ISSUE #37: As a librarian, I want to set the default loan period, renew period and fine amount
so that I can I customize the app to suit my library's policies

### LINKED ISSUE #72: Implement Set command and functionality

### PR DESCRIPTION
**Pull request by Cronyxx**
This closes #37 closes #72

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/UserSettings.java [hunk 1]
```diff
@@ -61,9 +60,9 @@ public boolean equals(Object other) {
 
         UserSettings o = (UserSettings) other;
 
-        return loanPeriod == o.loanPeriod
-                && renewPeriod == o.renewPeriod
-                && fineIncrement == o.fineIncrement;
+        return getLoanPeriod() == o.getLoanPeriod()
+                && getRenewPeriod() == o.getRenewPeriod()
+                && getFineIncrement() == o.getFineIncrement();
```
Comment (linyutinglyt) [inline src/main/java/seedu/address/commons/core/UserSettings.java:65]: Good refactoring!

**Review by linyutinglyt — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/model/usersettings/LoanPeriod.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.usersettings;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the default loan period of a book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLoanPeriod(String)}
+ */
+public class LoanPeriod {
```
Comment (seeziyang) [inline src/main/java/seedu/address/model/usersettings/LoanPeriod.java:10]: Great OOP style and SRP!

File: src/test/java/seedu/address/testutil/UserSettingsUtil.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * A utility class for UserSettings.
+ */
+public class UserSettingsUtil {
+    /**
+     * Returns the part of command string for the given {@code SetUserSettingsDescriptor}'s details.
+     */
+    public static String getSetUserSettingsDescriptorDetails(SetCommand.SetUserSettingsDescriptor descriptor) {
+        StringBuilder sb = new StringBuilder();
```
Comment (shiyao821) [inline src/test/java/seedu/address/testutil/UserSettingsUtil.java:17]: Good use of String Builder

**Review by seeziyang — COMMENTED** [review]
LGTM!

**Review by hoholyin — APPROVED** [review]
LGTM!

**Pull request merged**